OpenRA.Game/GameRules/ActorInfo.cs

```diff
@@ -38,17 +38,17 @@ public ActorInfo(ObjectCreator creator, string name, MiniYaml node)
 			{
 				Name = name;
 
-				var abstractActorType = name.StartsWith("^");
+				var traitCollectionNamePrefix = "^";
+				var ignoreMissingFields = name.StartsWith(traitCollectionNamePrefix);
```

Comment:
Trying to parse an `ActorInfo` from a partial template is bogus to begin with.  A tricker but IMO much cleaner fix would be to filter these out before trying to construct them in the `Ruleset`.

Reply:
Hmm, yes - this was a conservative change in that regard- I'm definitely still building up my mental model here.

If I'm interpreting correctly (a very big if, given my ignorance of the range of data shapes travelling here) you would like to recognize and pre-emptively filter the incoming list of Actors names in RuleSet.Load(), for generating the info's?

I'll do some quick prelim checks to get a sense on if that approach hits any obvious snags/problem cases.

Reply:
I've prepped a change based on suggestion above - if that approach looks sound, I'll drop the failure handling from here.
https://github.com/OpenRA/OpenRA/pull/14859